pinot-broker/src/main/java/org/apache/pinot/broker/broker/helix/BrokerUserDefinedMessageHandlerFactory.java

```diff
@@ -129,6 +136,32 @@ public void onError(Exception e, ErrorCode code, ErrorType type) {
     }
   }
 
+  private class RefreshDatabaseConfigMessageHandler extends MessageHandler {
```

Comment:
This helix msg only updates db quota on broker. But it is used for addDatabaseConfig controller side restful API. So when addDatabaseConfig, broker may not create the db quota. Is this expected behavior?

```
 public void addDatabaseConfig(DatabaseConfig databaseConfig) {
    if (!ZKMetadataProvider.createDatabaseConfig(_propertyStore, databaseConfig)) {
      throw new RuntimeException("Failed to create database config for database: " + databaseConfig.getDatabaseName());
    }
    sendDatabaseConfigRefreshMessage(databaseConfig.getDatabaseName());
  }
```

Reply:
An empty rate limiter is created for all existing tables/database on broker startup/table creation if no config is provided so there will always be a rate limiter entry in the map for the above message to update.